bridges/opentracing/lib/opentelemetry/bridge/opentracing/reference.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Bridge
+    module OpenTracing
+      # Reference provides a means of treating
+      # an OpenTelemetry::Link as an OpenTracing::Reference
+      class Reference
```

Comment:
This won't work because `OpenTracing::Reference` is effectively a concrete class -- users of OpenTracing will create instances directly rather than creating instances of our implementation.

Reply:
I switched from this to convenience methods `from_link` and `to_link`